web/server.js

```diff
@@ -260,6 +260,33 @@ module.exports = function(options) {
       path: '/create-user',
       config: {
         auth: false,
+        pre: [
+          {
+            assign: 'username',
+            method: function(request, reply) {
+              reply(request.payload.username);
+            }
+          },
+          {
+            assign: 'password',
+            method: function(request, reply) {
+              var password = request.payload.password;
+              if ( !password ) {
```

Comment:
we don't need this check, since we're guaranteed it'll be here by the payload validation config